dev/watch.js

```diff
@@ -96,6 +97,9 @@ chokidar.watch(`${sassDir}/**/*.scss`).on('change', changedFile => {
                 filesToCompile.map(file => file.replace('scss', 'css'))
             );
         })
+        .then(() => {
+            copyFiles.task();
```

Comment:
see here for an example https://github.com/guardian/frontend/blob/master/common/app/views/fragments/stylesheets.scala.html#L54-L60